test/extended/operators/operators.go

```diff
@@ -156,6 +182,56 @@ var _ = g.Describe("[Feature:Platform][Smoke] Managed cluster should", func() {
 			e2e.Failf("None of the required core operators are available")
 		}
 	})
+
+	g.It("should iterate through operator pods and detect higher-than-normal queries per second", func() {
```

Comment:
You aren't a smoke test, you don't belong in here.  Move to a separate describe.